.github/workflows/generate_workflows_lib/src/generate_workflows_lib/lint.yml.j2

```
@@ -10,7 +10,9 @@ on:
   pull_request:
 
 env:
+  CORE_REPO: open-telemetry/opentelemetry-python
   CORE_REPO_SHA: main
+  CONTRIB_REPO: open-telemetry/opentelemetry-python-contrib
```

Does this need to be`${{ github.repository }}` so users can run tests of contrib forks too? Just going off of symmetry here

---

hmm, it probably needs to be:
```suggestion
  CONTRIB_REPO: ${% raw %}{{ inputs.CONTRIB_REPO || 'open-telemetry/opentelemetry-python-contrib' }}{% endraw %}
```

---

@aabmass I think this is not needed, because checkout action already does the trick for us, including forks.

---

In principle, I believe, this could be needed if someone was trying to test a change to their opentelemetry-python repository that needed changes to their opentelemetry-python-contrib repository at the same time (which is more or less what we're looking at in https://github.com/open-telemetry/opentelemetry-python/pull/4388 where things conceptually didn't work for https://github.com/jsoref/opentelemetry-python/actions/runs/12793563878 because the callee doesn't yet have the required feature).

I'm not saying it's working, but I suspect a thing like it would be needed. I'm also not entirely certain it would always work/would work in this case (I think you can't really programmatically change a called workflow target, so in this specific case it wouldn't, but for a normal case, of code as opposed to workflow dependencies, I think it'd be useful).